src/Microsoft.ML.StandardLearners/Standard/SdcaBinary.cs

```diff
@@ -1523,20 +1523,6 @@ private protected SdcaBinaryTrainerBase(IHostEnvironment env, BinaryOptionsBase
 
         private protected abstract SchemaShape.Column[] ComputeSdcaBinaryClassifierSchemaShape();
 
-        private protected override void CheckLabelCompatible(SchemaShape.Column labelCol)
```

Comment:
>CheckLabelCompatible [](start = 40, length = 20)

Can label be text? If no, we still a check.

Reply:
As you can see this is override method. By removing this one, I would force system to fallback to default behavior. Which is - to check it's a boolean column.

---
In reply to: [263182006](https://github.com/dotnet/machinelearning/pull/2804#discussion_r263182006) [](ancestors = 263182006)